scene/3d/collision_object.h

```diff
@@ -94,6 +96,12 @@ friend class Viewport;
 	void set_capture_input_on_drag(bool p_capture);
 	bool get_capture_input_on_drag() const;
 
+	void set_center_of_mass(const Vector3& p_center_of_mass);
+	Vector3 get_center_of_mass() const;
+
+	Vector3 get_global_center_of_mass() const;
+
+	void calculate_center_of_mass();
```

Comment:
Must the user call this each time the collision shapes are moved or added/removed? IMO it is better to have the engine autmatically compute the center of mass when this happens